bin/interpreter/types/boolean.js

```diff
@@ -11,6 +11,7 @@ class SWBoolean extends SWValue {
   constructor(value) {
     super();
     this.value = value;
+    this.typeName = 'Hali';
```

Comment:
Booleans type is called Hali?

Reply:
I guess lol. Couldn't think of anything else

Reply:
smart